### DISCUSSION TITLE
Tutorial for working with times?

### DISCUSSION THREAD

**Discussion by zerok** [cue-lang/cue#1251] [Q&A] [answered]
Hi 🙂 

Sorry if this has already been answered before but I couldn't find anything about that yet. Is there perhaps a tutorial somewhere for working with the `time` module? I would like to create a definition for a struct looking somehow like this:

```
#Event: {
  createdAt: string
}
```

`createdAt` should be a valid date according to a given layout. Can this be done and if so how? Thank you so much! 🙂

**Comment by verdverm** [comment, marked as answer]
Not necessarily a tutorial on time, but here are two examples

https://cuetorials.com/overview/standard-library/ (at the bottom)

https://cuetorials.com/first-steps/validate-configuration/ (custom format, date only)

**Reply by zerok** [reply]
Awesome! Thank you so much!
